### DISCUSSION TITLE
Ability to Quickly Select Different Payment Methods by Clicking on the Label

### DISCUSSION THREAD

**Discussion by nara-golfdistrict-com** [juspay/hyperswitch#3713] [Ideas / Feature Requests]
When I am selecting a new processor, I need to select the different payment methods by selecting the different checkboxes. Today I have to select the actual checkbox which requires precision. 
Can you please allow us to select the label (Like Visa or Master Card), so that we can quickly select different options?
See below screenshot.
![image](https://github.com/juspay/hyperswitch/assets/145275696/85d98053-2039-4dcd-83f6-9cdc7f6f3cd1)

**Comment by gitanjli525** [comment]
Sure @nara-golfdistrict-com , same has been added as feature. 
Refer the [issue](https://github.com/juspay/hyperswitch-control-center/issues/431).

Thank you

**Comment by nara-golfdistrict-com** [comment]
Thanks Gitanjli for taking care of this quickly. Much appreciated.

From: Gitanjli ***@***.***>
Date: Thursday, February 22, 2024 at 4:39 AM
To: juspay/hyperswitch ***@***.***>
Cc: Nara Sankar ***@***.***>, Mention ***@***.***>
Subject: Re: [juspay/hyperswitch] Ability to Quickly Select Different Payment Methods by Clicking on the Label (Discussion #3713)

Sure @nara-golfdistrict-com<https://github.com/nara-golfdistrict-com> , same has been added as feature.
Refer the issue<https://github.com/juspay/hyperswitch-control-center/issues/431>.

Thank you
